pom.client.xml

```diff
@@ -1247,6 +1247,7 @@
     <module>sdk/storage/azure-storage-file-share</module>
     <module>sdk/storage/azure-storage-file-datalake</module>
     <module>sdk/storage/azure-storage-queue</module>
+    <module>sdk/search/azure-search</module>
```

Comment:
nit: the rest of the modules are in alphabetical order, please put **search** above **storage**.